src/components/Text/TextEffects.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import keyMirror from 'keymirror';
+import { helpers } from '../../utils/';
+
+const TEXT_TAGS = keyMirror({
+  condensed: null,
+});
+
+const CondensedText = ({
+  children,
+  className,
+  condenseThreshold = 0,
+  enableCopyIcon = false,
```

Comment:
This is just a small nit, but how about just `copyIcon` without "enable". `copyIcon: true` basically says the same thing and it makes it a little more compact.

Reply:
re: nit

copyIcon namespace is already taken by the destructuring of the glyph theme.

Reply:
How about we do this to make the namespace available for your nit:

``` theme: { text: { iconActive, iconInactive }, glyphs }```
```        className={`fa fa-copy ${glyphs.copyIcon}`}```